README.md

```diff
@@ -73,16 +73,16 @@ fn main() {
 
 ## Let there be SparseSets
 
-I initially started to make an ECS to learn how it works. A failed attempt and some research later, I started to work on Shipyard.
+I initially started to make an ECS to learn how they work. After a failed attempt and some research, I started to work on Shipyard.
 
 [Specs](https://github.com/amethyst/specs) was already well established as the go-to Rust ECS but I thought I could do better and went with [EnTT](https://github.com/skypjack/entt) core data-structure: `SparseSet`.
 
-It's extremely flexible and is the core data-structure behind Shipyard.  
-Today I wouldn't say Shipyard is better or worse than Specs, it's just different.
+It's extremely flexible and is the core data structure behind Shipyard.  
+I wouldn't say Shipyard is better or worse than Specs, it's just different.
 
 ## Systems
 
-Systems make it very easy to split your logic in manageable chunk. Shipyard takes the concept quite far.
+Systems makes it very easy to split your logic in manageable chunk. Shipyard takes the concept quite far.
```

Comment:
This is a tricky one, I'm not actually sure. Gotta consult Strunk and White :)

My gut feeling tells me Systems is plural so it's `make`, but on the other hand `Systems` is itself almost a proper name here, almost like Cheerios. I'd like towards `make` but could be wrong.

Reply:
I think that "makes" would technically be right since it should be read as "[the idea of] systems makes it very easy...", i.e. it's not the systems themselves that make it easy, it's the act of thinking in terms of systems. Although "make" sounds better to me and I think that's reasonable too 🤷‍♂️

Also, s/chunk/chunks/ later in the sentence 🙂

Reply:
Sorry – I think this change should be reverted (make not makes). I don't think `Systems` is a proper noun.

Reply:
```suggestion
Systems make it very easy to split your logic in manageable chunks. Shipyard takes the concept quite far.
```